### ISSUE TITLE
How to design the Multiple softmax output layers of the network, where there is a corresponding case, I always get error failure.

### ISSUE THREAD

**Issue by dlunion** [tiny-dnn/tiny-dnn#676]
How to design the Multiple softmax output layers of the network, where there is a corresponding case, I always get error failure.  3qs.

**Comment by ljluestc** [comment]
```

import tensorflow as tf
from tensorflow.keras.layers import Input, Dense
from tensorflow.keras.models import Model

# Define input layer
inputs = Input(shape=(input_shape,))

# Define shared hidden layers
shared_hidden1 = Dense(64, activation='relu')(inputs)
shared_hidden2 = Dense(32, activation='relu')(shared_hidden1)

# Define separate output layers for each case
output1 = Dense(num_classes1, activation='softmax', name='output1')(shared_hidden2)
output2 = Dense(num_classes2, activation='softmax', name='output2')(shared_hidden2)
output3 = Dense(num_classes3, activation='softmax', name='output3')(shared_hidden2)

# Define the model with input and output layers
model = Model(inputs=inputs, outputs=[output1, output2, output3])

# Compile the model
model.compile(optimizer='adam', loss='categorical_crossentropy', metrics=['accuracy'])

# Train the model
model.fit(x_train, [y_train1, y_train2, y_train3], epochs=10, batch_size=32)

# Evaluate the model
loss, acc1, acc2, acc3 = model.evaluate(x_test, [y_test1, y_test2, y_test3])
```
